### DISCUSSION TITLE
On Psycho Gas

### DISCUSSION THREAD

**Discussion by Moburma** [swfans/syndwarsfx#215] [General]
Don't think I've ever seen anything written about how this actually works in the game. Some findings from testing it:

EDIT: Point 3 below ended up being more nuanced than I first thought...

- On deployment, the grenade expels gas for about 12 seconds
- The effect of the gas seems to work somewhere between 15 - 20 seconds. It's hard to say how long as being exposed to more gas seems to reset the timer
- As above I _believe_ being in the gas resets the timer for how long it lasts for, so if they are inside the cloud they will keep being psychotic for longer
- Once a peep is psychotic, they erratically run about and will attack other peeps for the duration of the psychotic period. The effect seems to follow the following rules:
1. They will randomly attack and kill peeps in other groups
2. They will randomly attack peeps in other groups they are truced with, but they cannot actually hit them, their attacks will always miss
3. They will attack peeps in the same group, but only if _they are not truced against their own group_. I always thought this was a weird oversight/mistake in levels (why would you need to truce a group against itself?) but this is why - if they are not truced against their own group they **will** kill each other when Psycho Gassed. 
4. Cannot cause long term inter-group combat. Two neutral groups who shoot/kill each other will not continue to be antagonised and fight each other after the gas effect has run out, in the way you can cause inter-group combat by making two neutral groups shoot each other outside of a gas attack.

2 Above is quite a major problem for the use of the gas by the player, as there is no way to know which peeps are in which groups. and which groups are truced or not. Also most allied NPCs _are_ truced anyway. This means that Psycho Gas is pretty useless in much of single player mode, unless you have clearly non-aligned groups (e.g. police vs zealots), but even that is hardly guaranteed.

**Comment by geist22** [comment]
One weird thing I noticed is that sometimes Psycho Gas prevents enemies from dropping their weapons when killed. For example, in EuroCorp Mission 9 (New York), there's a temple with a Psycho Gas trap and Zealots that drop an Electron Mace and Disruptor each. But they will only drop their weapons if you kill them without them getting hit by the gas. If they get affected, they'll drop nothing upon death.

I couldn't get it to work with my own Psycho Gas in other places, so maybe that's a thing specific to the traps or the enemies there.
